readme.md

```diff
@@ -48,6 +50,24 @@ bind -n M-o run 'harpoon -s 3'
 bind -n M-s run 'harpoon -s 4'
 ```
 
+To map panes or session to a specific index, use the -o/O flag:
+
+```conf
+bind -n M-q run 'harpoon -s 1'  # alt-q goes to index 1
```

Comment:
we can merge this example tmux configuration with the one above. add shorter comments at end of line to explain what the configuration is about. At the same time make the keybinds consistent. I like your approach of have qwerty denote the position.

Reply:
Make keybinds consistent, as in change the existing binds to qwerty?